src/Components/LetterBuilder/FormSteps/ConfirmationStep.tsx

```diff
@@ -75,8 +74,8 @@ export const ConfirmationStep: React.FC<{
           </p>
           <p>
             <Trans>
-              You can try again or contact our support team at
-              support@justfix.org if the issue continues.{" "}
+              You can try again or contact our support team at{" "}
+              <strong>support@justfix.org</strong> if the issue continues.{" "}
```

Comment:
can't remember if we discussed making this a mailto link like we have on confirmation page error notice
```suggestion
              <a target="_blank"
              rel="noopener noreferrer" 
              className="jfcl-link" 
              href="mailto:support@justfix.org?subject=Good%20Cause%20NYC%20Letter">
              <strong>support@justfix.org</strong>
              </a> if the issue continues.{" "}
```